doc/api/os.md

```diff
@@ -215,6 +215,14 @@ added: v2.3.0
 The `os.homedir()` method returns the home directory of the current user as a
 string.
 
+POSIX:
+Will use the `$HOME` environment variable if defined. Otherwise, it will use
+the effective user home directory.
```

Comment:
Sr yes sr! Would you recommend me a confident font, a respected one? I mean, I won't quote Wikipedia

Reply:
https://en.wikipedia.org/wiki/User_identifier#Effective_user_ID is actually pretty good, IMO. It's clear on that the EUID is what is used for most access checks and from that it follows why `os.homedir()` would use the effective rather than the real UID. 

https://stackoverflow.com/a/32456814 is also okay.